cypress/support/commands.js

```diff
@@ -45,24 +46,48 @@ Cypress.Commands.add('login', (email, username, password) => {
   });
 });
 
-Cypress.Commands.add('createArticle', (title, description, body) => {
-  cy.getCookie('auth').then((token) => {
-    const authToken = token.value;
-
-    cy.request({
-      method: 'POST',
-      url: '/api/articles',
-      body: {
-        article: {
-          title,
-          description,
-          body,
-          tagList: []
-        }
-      },
-      headers: {
-        Authorization: `Token ${authToken}`
+Cypress.Commands.add('createArticle',
+  (username, email, password, title, description, body) => {
```

Comment:
The 'createArticle' function is now also handling user registration, which may not be desirable as it combines two functionalities. Consider separating user registration and article creation into different functions for better separation of concerns.

Reply:
No need to modify given commands, you could just reuse it as a ready solution.